Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/civimed.yml

```
@@ -15,3 +15,4 @@
     BikeHornImplanterLabeled: 4294967295 # Infinite
     LightImplanterLabeled: 4294967295 # Infinite
     MedTekCartridge: 4294967295 # Infinite
+    BoxFolderMedicalDoctor: 4294967295 # Frontier - infinite
```

```suggestion
    BoxFolderMedicalDoctor: 4294967295 # Infinite
```
_NF namespaced file, no Frontier needed